src/TuDa.CIMS.Web/Services/IPurchaseApi.cs

```diff
@@ -33,7 +33,7 @@ public async Task<ErrorOr<Purchase>> GetAsync(Guid workingGroupId, Guid id) =>
     /// A Task representing the asynchronous operation, with a result of <see cref="IApiResponse{T}"/> containing the Purchase.
     /// </returns>
     /// <remarks>This will be generated by <see cref="Refit"/>.</remarks>
-    [Get("/{workingGroupId}/{id}")]
+    [Get("/{id}")]
```

Comment:
🔧 It needs to be like this
```suggestion
    [Get("/{workingGroupId:guid}/purchases/{id}")]
```